### ISSUE TITLE
Add support for utoken tokenizer

### ISSUE THREAD

**Issue by ddaspit** [sillsdev/machine.py#2]
[utoken](https://github.com/uhermjakob/utoken) is a general-purpose word tokenizer, in the spirit of sacremoses. Machine can provide a utoken implementation of the tokenizer interface.

**Comment by johnml1135** [comment]
@ddaspit - is this still something we want to do?  If not, let's close the issue.

**Comment by ddaspit** [comment]
Yes, we still want to do this at some point. It would be done to make it available in the library.
